Gemfile.lock

```diff
@@ -13,11 +13,13 @@ GEM
     csv (3.2.8)
     diff-lcs (1.5.1)
     docile (1.4.0)
+    equivalent-xml (0.6.0)
+      nokogiri (>= 1.4.3)
     json (2.7.1)
     language_server-protocol (3.17.0.3)
-    mini_portile2 (2.8.1)
-    nokogiri (1.13.10)
-      mini_portile2 (~> 2.8.0)
+    nokogiri (1.13.10-arm64-darwin)
+      racc (~> 1.4)
+    nokogiri (1.13.10-x86_64-linux)
```

Comment:
Have to keep this platform independent.

Reply:
@ronaldtse I think there should be no `Gemfile.lock` present in the gem. This file was present in the original gem.
For now I'll revert the changes.